notebooks/220-license-plate-recognition/220-license-plate-recognition.ipynb

```diff
@@ -0,0 +1,483 @@
+    ")\n",
+    "\n",
+    "# this code is provided for the first download of the model\n",
+    "! $download_command"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load the Model\n",
+    "\n",
+    "First, we create an Inference Engine. Then we read the network architecture and model weights from the .bin and .xml files to compile for the desired device. You can choose manually CPU, GPU, MYRIAD etc.\n",
+    "\n",
+    "If you wnat OpenVINO to decide which hardware offers the best performance, you need to use `AUTO`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "# \"FP16\", \"FP32\" or \"FP16-INT8\"\n",
+    "precision = \"FP16\"\n",
+    "# output path for the conversion\n",
+    "converted_model_path = f\"model/intel/{model_name}/{precision}/{model_name}.xml\"\n",
+    "\n",
+    "# initialize inference engine\n",
+    "ie_core = Core()\n",
+    "# read the network and corresponding weights from file\n",
+    "model = ie_core.read_model(model=converted_model_path)\n",
+    "# compile the model for the CPU (you can choose manually CPU, GPU, MYRIAD etc.)\n",
+    "# or let the engine choose the best available device (AUTO)\n",
+    "compiled_model = ie_core.compile_model(model=model, device_name=\"CPU\")\n",
+    "\n",
+    "# get input and output nodes\n",
+    "input_layer = next(iter(compiled_model.inputs))\n",
+    "output_layer = next(iter(compiled_model.outputs))\n",
+    "\n",
+    "# get input size\n",
+    "input_height, input_width = list(input_layer.shape)[2:4]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    },
+    "tags": []
+   },
+   "source": [
+    "## Inference\n",
+    "\n",
+    "### Using ImageLoader to Process Image\n",
+    "\n",
+    "\n",
+    "#### Model Limitations:\n",
+    "Only \"blue\" license plates, which are common in public, were tested thoroughly. Other types of license plates may underperform.\n",
+    "\n",
+    "\n",
+    "#### Model Input:\n",
+    "\n",
+    "name: \"data\" , shape: [1x3x24x94] - An input image in following format [1xCxHxW]. Expected color order is BGR.\n",
+    "\n",
+    "name: \"seq_ind\" , shape: [88,1] - An auxiliary blob that is needed for correct decoding. Set this to [0, 1, 1, ..., 1].\n",
+    "\n",
+    "#### Notes:\n",
+    "Since the license plate image could be any size, we need to use ImageLoader to resize it to fit the model input requirements.\n",
+    "Augmentation method on image is optional and it may lead to the wrong recognition result. Thus, we recommend you to use it under special conditions like image overexposure or too dark."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def ShowImage(\n",
```

Comment:
Please change function and variable names as stated in https://peps.python.org/pep-0008/#function-and-variable-names